src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length <= 1) {
```

Comment:
Checking for a string length of less than or equal to 1 might not be the best way to determine if the input string is empty or not valid. Consider checking for an empty string after trimming for whitespace.